homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,89 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: todo
+    comment: Add tests preventing second entry for same device
+  config-flow-test-coverage:
+    status: todo
+    comment: Cover test_form and zeroconf
+  runtime-data:
+    status: todo
+    comment: Clean up test_init for testing internals
+  test-before-setup: done
+  appropriate-polling:
+    status: todo
+    comment: Clean up coordinator (L71) check for mypy happiness
+  entity-unique-id: done
+  has-entity-name:
+    status: todo
+    comment: Clean up climate (already in superclass)
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: todo
+    comment: Check if we have these, otherwise exempt
+  common-modules:
+    status: todo
+    comment: Verify entity for async_added_to_hass usage (discard?)
+  docs-high-level-description: done
+  docs-installation-instructions: done
```

Comment:
I am missing the default config flow setup snippet and I personally think the text is a bit too literal. I think there is room for improvement here.